benchmark/index.js

```diff
@@ -87,6 +92,42 @@ test('setup', (t) => {
   t.end()
 })
 
+test('add a bunch of messages', async (t) => {
+  const sbot = SecretStack({ appKey: caps.shs })
+    .use(require('../'))
+    .call(null, { keys, path: dirAdd })
+
+  let state = validate.initial()
+  for (var i = 0; i < 1000; ++i) {
+    state = validate.appendNew(state, null, keys, { type: 'tick', count: i }, Date.now())
+  }
+
+  const ended = DeferredPromise()
+  const start = Date.now()
+
+  pull(
+    pull.values(state.queue.map(x => x.value)),
+    asyncFilter(sbot.db.add),
+    pull.collect((err) => {
+      if (err) t.fail(err)
+
+      const duration = Date.now() - start
```

Comment:
The other benchmarks have the `duration` declaration before the `if (err) t.fail(err)`, to avoid measuring the `if`. I know, the `if` lasts 1 nanosecond maximum, but just for consistency with the other code, and since you're changing code here, might as well just move it